gqlalchemy/memgraph.py

```diff
@@ -543,3 +544,26 @@ def create_relationship(self, relationship: Relationship) -> Optional[Relationsh
         )
 
         return self.get_variable_assume_one(results, "relationship")
+
+    def get_procedures(self, starts_with: Optional[str] = None, update: bool = False) -> List["QueryModule"]:
+        """Return query procedures.
+
+        Maintains a list of query modules in the Memgraph object. If starts_with
+        is defined then return those modules that start with starts_with string.
+
+        Args:
+            starts_with: Return those modules that start with this string.
+            (Optional)
+            update: Whether to update the list of modules in
+            self.query_modules. (Optional)
+        """
+        if not hasattr(self, "query_modules") or update:
+            results = self.execute_and_fetch("CALL mg.procedures() YIELD *;")
+            self.query_modules = []
```

Comment:
rewrite this as list comprehension

Reply:
self.query_modules = [QueryModule(x) for x in results] I guess